gister.sh

```diff
@@ -308,7 +308,13 @@ sync_gist() {
           legit sync > /dev/null
         else
           if (is_dirty); then
-            echo "DIRTY $gist_id"
+            if [ -z $GISTER_AUTO_COMMIT ]; then
+              echo "DIRTY $gist_id"
+            else
+              echo "DIRTY $gist_id, auto committing"
+              git add . && git commit -m "automated update" && \
```

Comment:
`&& \` is unnecessary. Just use `&&`.

Reply:
Good catch, thanks!

Reply:
I was meaning to escape the end of line character which I thought was necisary, but it's probably best to have them as separate lines anyway.